.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
 script:
   - "python setup.py sdist"
   - "pip install ."
+  - "pip install .[dev]"
```

Comment:
You can remove the line above, as `.[dev]` includes `.`  by definition.

Reply:
That feels weird to me but makes sense with our abuse of the extra system in this way.  We're saying install . with dev and not the dev for ..  I'm inclined to leave it since it should be a noop or trivial duplication for the sake of making things more explicit.